config/class-site-details-index.php

```diff
@@ -145,9 +145,9 @@ public function get_jetpack_info() {
 
 		if ( class_exists( 'Jetpack' ) ) {
 			$jetpack_info['available'] = true;
-			$jetpack_info['active'] = \Jetpack::is_active();
-			$jetpack_info['id'] = \Jetpack::get_option( 'id' );
-			$jetpack_info['version'] = JETPACK__VERSION;
+			$jetpack_info['active']    = \Jetpack::is_active();
```

Comment:
**WARNING:** This method is deprecated. I am not familiar with Jetpack good enough to fix it now.